addons/jest.js

```diff
@@ -10,4 +10,23 @@ module.exports = {
     'plugins': [
         'jest',
     ],
+    'overrides': [{
+        'env': {
+            'jest/globals': false,
+        },
+        'files': [
+            '**/*.js',
+        ],
+        'excludedFiles': [
+            '**/__tests__/**/*.js',
+            '**/?(.*)*@(spec|test).js',
+            'test/**/*',
+        ],
+        'rules': {
```

Comment:
Get the rules defined in `rules/jest.js` and apply a map operation to deactivate them all

Reply:
https://github.com/jest-community/eslint-plugin-jest/blob/80c9b33e5e44529afb01d38c392b17bb1deee0ea/index.js#L65